.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you remove this?

Reply:
I'm sorry that I made many mistakes. I deleted it.